tests/by-util/test_chroot.rs

```diff
@@ -64,6 +64,28 @@ fn test_invalid_user_spec() {
     assert!(result.stderr_str().starts_with("chroot: invalid userspec"));
 }
 
+#[test]
+fn test_invalid_user() {
+    let ts = TestScenario::new(util_name!());
+    let at = &ts.fixtures;
+
+    let dir = "CHROOT_DIR";
+    at.mkdir(dir);
+    if let Ok(result) = run_ucmd_as_root(&ts, &[dir, "whoami"]) {
+        result.success().no_stderr().stdout_is("root");
+    } else {
+        print!("Test skipped; requires root user");
+    }
+
+    if let Ok(result) = run_ucmd_as_root(&ts, &["--user=nobody:+65535", dir, "pwd"]) {
+        result
+            .failure()
+            .stderr_contains("no such user: nobody:+65535");
```

Comment:
Is the error message deliberately different from the one of GNU `chroot`?

Reply:
yeah, it is for a next step
I am just removing the panic